ts/features/services/screens/ServiceDetailsScreen.tsx

```diff
@@ -124,6 +133,13 @@ const ServiceDetailsContent = ({ service }: ServiceDetailsContentProps) => {
           <VSpacer size={16} />
         </ContentWrapper>
       </View>
+      {service.service_metadata?.description ? (
```

Comment:
You could insert the logical and (&&) instead of the ternary operator (? and : )

Reply:
Done it https://github.com/pagopa/io-app/pull/5640/commits/ec1c594e7ed6e408cc30fb017010afef0135a779